### DISCUSSION TITLE
Does future/current imagemagick develop will support for stl(STereoLithography) file(3d image file) format creation or any other such freeware exist? Thanks in advance for guiding

### DISCUSSION THREAD

**Discussion by abilask** [ImageMagick/ImageMagick#5502] [General]
Hi Fiends, 

Does future/current imagemagick develop will support for stl(STereoLithography) file(3d image file) format creation or any other such freeware exist? Thanks in advance for guiding

**Comment by snibgo** [comment]
According to https://en.wikipedia.org/wiki/STL_%28file_format%29 , STL is a simple 3D CAD format that records the vertexes and normals of triangles.

ImageMagick is a processor for 2D raster image files, which are entirely different to 3D CAD.

I suggest Blender as a possibility.
